ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -83,4 +58,89 @@ public extension UIColor {
             }
         }
     }
+
+    /// Creates a UIColor from a single 32-bit integer value.
+    ///
+    /// - Parameter hexValue: Integer value, generally represented as hexadecimal (e.g. `0x0086F0`), to use to initialize this color.
+    ///     Must be formatted as ARGB. Note: we will not utilize the alpha channel.
+    convenience init(colorValue: ColorValue) {
+        self.init(
+            red: CGFloat((colorValue & 0xFF0000) >> 16) / 255.0,
+            green: CGFloat((colorValue & 0x00FF00) >> 8) / 255.0,
+            blue: CGFloat((colorValue & 0x0000FF) >> 0) / 255.0,
+            alpha: 1.0)
```

Comment:
Please remind me: Why don't we support alpha in the ColorValue?

Reply:
Limitation of the design token pipeline. Colors are pure web-style 24-bit hex, with alpha channels stored as separate tokens.

Reply:
I'll have to discuss with design; right now their intent is for opacity to be stored separately from the color, which should be sufficient for our needs. Let's sync offline to discuss the pros and cons of such an approach.